File: service/danmakuservice/src/main/java/com/qlive/danmakuservice/QDanmakuServiceListener.java
package com.qlive.danmakuservice;

import com.qlive.core.been.QDanmaku;

/**
 *弹幕消息监听
 */
public interface QDanmakuServiceListener {
    /**
     * 收到弹幕消息
     * @param danmaku 弹幕实体
     */
    void onReceiveDanmaku(QDanmaku danmaku);
}

File: service/danmakuservice/src/main/java/com/qlive/danmakuservice/QDanmakuServiceListener.kt
package com.qlive.danmakuservice

import com.qlive.core.been.QDanmaku

/**
 *弹幕消息
 */
interface QDanmakuServiceListener {
    /**
     * 收到弹幕消息
     * @param danmaku 弹幕实体
     */
    fun onReceiveDanmaku(danmaku: QDanmaku)
}